src/Authentication.Abstractions/Authentication.Abstractions.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <IncludeHyak>true</IncludeHyak>
+    <IsCmdletProject>false</IsCmdletProject>
```

Comment:
@isral-fel, Is it still required?

Reply:
I'm not 100% sure if false = not specified, so let me just revert this csproj file.